.github/workflows/e2e-tests.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v2
         with:
-          start: yarn dev
+          start: CI=true yarn dev
```

Comment:
Would it be better to set `CI=true` for the whole workflow rather than just this one command?

Reply:
Maybe yeah. I'm stabbing in the dark. :/

Reply:
Its already set. :(

Reply:
So it is.